### ISSUE TITLE
nfssec.properties is missing in the jar artifact

### ISSUE THREAD

**Issue by karayv** [raisercostin/yanfs#3]
I'm getting this exception while trying to use the library:
```java
java.lang.NullPointerException
	at com.sun.nfs.NfsSecurity.getMech(NfsSecurity.java:198)
	at com.sun.nfs.NfsConnect.tryNfs(NfsConnect.java:328)
	at com.sun.nfs.NfsConnect.connect(NfsConnect.java:288)
	at com.sun.nfs.NfsConnect.connect(NfsConnect.java:113)
	at com.sun.nfs.XFileAccessor.open(XFileAccessor.java:70)
	at com.sun.xfile.XFile.bind(XFile.java:409)
	at com.sun.xfile.XFile.canRead(XFile.java:701)
//... skipped
```
The root cause is missing `nfssec.properties`. If I manually put the file in my source code inside package `com.sun.nfs`, the library works.

**Comment by Caiws** [comment]
hi，can you update the jar? I download from mvnrepository, still missing properties.

**Comment by raisercostin** [comment]
Released yanfs-1.4 with `nfssec.properties` file included.
